.github/workflows/ci.yml

```
@@ -15,6 +15,9 @@ jobs:
         node-version: [10.x, 12.x, 14.x]
 
     steps:
+      - name: semantic-pull-request
+        uses: amannn/action-semantic-pull-request@v3.4.2
```

```suggestion
      - name: semantic-pull-request
        uses: amannn/action-semantic-pull-request@v3.4.2
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

env.GITHUB_TOKEN을 제공해줘야하는 것 같습니다.!
해당 액션 코드 확인 결과 PR 정보를 얻어오기위해 깃헙토큰 정보를 활용하네요 ㅎㅎ [액션 src/index.js 링크](https://github.com/amannn/action-semantic-pull-request/blob/master/src/index.js)
```javascript
// action-semantic-pull-request/src/index.js 코드 스니펫입니당
module.exports = async function run() {
  try {
    const client = github.getOctokit(process.env.GITHUB_TOKEN);
```

---

@greatSumini 옷, 그렇네요. 
@milooy https://github.com/amannn/action-semantic-pull-request#example-config 관련 예제 참고하셔도 좋을 것 같습니다.

---

그러네요! 감사합니다 ㅋㅋ 바로 commit!

---

env.GITHUB_TOKEN 은 깃헙이 자동으로 넣어주는 secret이네요 ([참고](https://docs.github.com/en/actions/reference/authentication-in-a-workflow))

push, pull_request 모두에서 돌고있어서 pull request에만 돌도록 바꾸었습니당